applications/nrf5340_audio/src/audio/streamctrl_broadcast_sink.c

```diff
@@ -584,17 +448,80 @@ static int zbus_subscribers_create(void)
 		return ret;
 	};
 
-	content_control_msg_sub_thread_id = k_thread_create(
-		&content_control_msg_sub_thread_data, content_control_msg_sub_thread_stack,
-		CONFIG_CONTENT_CONTROL_MSG_SUB_STACK_SIZE,
-		(k_thread_entry_t)content_control_msg_sub_thread, NULL, NULL, NULL,
-		K_PRIO_PREEMPT(CONFIG_CONTENT_CONTROL_MSG_SUB_THREAD_PRIO), 0, K_NO_WAIT);
-	ret = k_thread_name_set(content_control_msg_sub_thread_id, "CONTENT_CONTROL_MSG_SUB");
+	return 0;
+}
+
+/**
+ * @brief	Zbus listener to receive events from bt_mgmt.
+ *
+ * @note	Will in most cases be called from BT_RX context,
+ *		so there should not be too much processing done here.
+ */
+static void bt_mgmt_evt_handler(const struct zbus_channel *chan)
+{
+	int ret;
+	const struct bt_mgmt_msg *msg;
+
+	msg = zbus_chan_const_msg(chan);
+	uint8_t event = msg->event;
+
+	if (event == BT_MGMT_PA_SYNC_OBJECT_READY) {
```

Comment:
```suggestion
	if (msg->event == BT_MGMT_PA_SYNC_OBJECT_READY) {
```

Reply:
This is probably true for all the other cases of zbus usage in this file and in streamctrl_broadcast_source

Reply:
the variable event is already msg->event (see line 466)